rules/windows/builtin/windefend/win_defender_disabled.yml

```diff
@@ -5,7 +5,8 @@ description: Detects disabling Windows Defender threat protection
 references:
     - https://docs.microsoft.com/en-us/windows/security/threat-protection/windows-defender-antivirus/troubleshoot-windows-defender-antivirus
     - https://github.com/redcanaryco/atomic-red-team/blob/f339e7da7d05f6057fdfcdd3742bfcf365fee2a9/atomics/T1562.001/T1562.001.md
-author: Ján Trenčanský, frack113
+    - https://www.microsoft.com/en-us/security/blog/2023/04/11/guidance-for-investigating-attacks-using-cve-2022-21894-the-blacklotus-campaign/
+author: Ján Trenčanský, frack113, Christopher Peacock '@securepeacock' (Update)
 date: 2020/07/28
 modified: 2022/12/06
```

Comment:
```suggestion
modified: 2023/05/01
```

Reply:
No need for this, I will will replace this addition by the one we have internally.